proxy/http3/Http3Frame.h

```diff
@@ -75,18 +75,17 @@ class Http3DataFrame : public Http3Frame
 public:
   Http3DataFrame() : Http3Frame() {}
   Http3DataFrame(const uint8_t *buf, size_t len);
-  Http3DataFrame(ats_unique_buf payload, size_t payload_len);
+  Http3DataFrame(IOBufferReader *reader, size_t payload_len);
 
-  void store(uint8_t *buf, size_t *len) const override;
+  int64_t store(QUICStreamIO *stream_io) override;
   void reset(const uint8_t *buf, size_t len) override;
 
-  const uint8_t *payload() const;
+  IOBufferReader *payload();
```

Comment:
Currently the payload return a new reader which is allocated from buffer. But we don't have the  deallocate function. So there is a limitation that we can only allocate 5 readers.